frontend/src/pages/PropertyDetails.jsx

```
@@ -0,0 +1,552 @@
+            </div>
+        );
+    }
+
+    return (
+        <div className="relative group">
+            {/* Main Image Container */}
+            <div className="relative w-full h-[500px] overflow-hidden rounded-lg">
+                {/* Main Image */}
+                <img 
+                    src={`http://localhost:4000/${images[currentImageIndex]}`} 
+                    alt={`Property image ${currentImageIndex + 1}`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)